inference/inference_benchmark/cc/Paddle/bin/run_models_benchmark.sh

```diff
@@ -10,7 +9,7 @@ export DATA_ROOT=$ROOT/Data
 export CASE_ROOT=$ROOT/bin
 export LOG_ROOT=$ROOT/log
 export gpu_type=`nvidia-smi -q | grep "Product Name" | head -n 1 | awk '{print $NF}'`
-
+source $ROOT/bin/run_clas_mkl_benchmark.sh
```

Comment:
why here need to source `run_clas_mkl_benchmark.sh`? is it only for tests

Reply:
User could run `bash bin/run_models_benchmark.sh "static" "cpu" "1 2 3" "1 2 3"` to run with batch_size 1, 2, 3, threads 1, 2, 3
or `bash bin/run_models_benchmark.sh "static" "cpu" "1" "1"` run with batch_size 1, threads 1